nengo_gui/static/netgraph.js

```diff
@@ -335,7 +335,7 @@ Nengo.NetGraph.prototype.on_message = function(event) {
         var uid = data.uid;
         for (var i = 0; i < Nengo.Component.components.length; i++) {
             if (Nengo.Component.components[i].uid === uid) {
-                Nengo.Component.components[i].remove(true);
+                Nengo.Component.components[i].remove(true, data.report_back);
```

Comment:
`report_back` seems like a bit of an ambiguous variable name. Is the only use of this variable to check whether the item was being undone or deleted? Is it associated to a particular event and is it likely that it will always be associated to that event?